app/src/main/java/com/duckduckgo/app/trackerdetection/blocklist/BlockListInterceptorApiPlugin.kt

```diff
@@ -14,67 +14,24 @@
  * limitations under the License.
  */
 
-package com.duckduckgo.app.trackerdetection
+package com.duckduckgo.app.trackerdetection.blocklist
 
-import com.duckduckgo.anvil.annotations.ContributesRemoteFeature
 import com.duckduckgo.app.global.api.ApiInterceptorPlugin
-import com.duckduckgo.app.trackerdetection.BlockList.Cohorts.CONTROL
-import com.duckduckgo.app.trackerdetection.BlockList.Cohorts.TREATMENT
-import com.duckduckgo.app.trackerdetection.BlockList.Companion.CONTROL_URL
-import com.duckduckgo.app.trackerdetection.BlockList.Companion.EXPERIMENT_PREFIX
-import com.duckduckgo.app.trackerdetection.BlockList.Companion.NEXT_URL
-import com.duckduckgo.app.trackerdetection.BlockList.Companion.TREATMENT_URL
 import com.duckduckgo.app.trackerdetection.api.TDS_BASE_URL
+import com.duckduckgo.app.trackerdetection.blocklist.BlockList.Cohorts.CONTROL
+import com.duckduckgo.app.trackerdetection.blocklist.BlockList.Cohorts.TREATMENT
+import com.duckduckgo.app.trackerdetection.blocklist.BlockList.Companion.CONTROL_URL
+import com.duckduckgo.app.trackerdetection.blocklist.BlockList.Companion.NEXT_URL
+import com.duckduckgo.app.trackerdetection.blocklist.BlockList.Companion.TREATMENT_URL
 import com.duckduckgo.di.scopes.AppScope
 import com.duckduckgo.feature.toggles.api.FeatureTogglesInventory
-import com.duckduckgo.feature.toggles.api.Toggle
-import com.duckduckgo.feature.toggles.api.Toggle.State.CohortName
 import com.squareup.anvil.annotations.ContributesMultibinding
 import javax.inject.Inject
 import kotlinx.coroutines.runBlocking
 import okhttp3.Interceptor
 import okhttp3.Interceptor.Chain
 import okhttp3.Response
 
-@ContributesRemoteFeature(
```

Comment:
This is me moving the interceptor to a different place and removing the BlockList feature form the same file.